repository/SmalltalkCI-Core.package/SmalltalkCI.class/instance/isCoverageTestingEnabled.st

```diff
@@ -1,5 +1,3 @@
 accessing
 isCoverageTestingEnabled
-	^ SmalltalkCI platformSymbol ~= #gemstone and:
-		[ (SmalltalkCI isTravisBuild or: [ SmalltalkCI isAppVeyorBuild ]) and:
-			[ self spec coverageEnabled ] ]
\ No newline at end of file
+	^ SmalltalkCI platformSymbol ~= #gemstone and: [ self spec coverageEnabled ]
```

Comment:
I've removed the extra travis/appveyor checks too, as they are no longer needed (if we consider has coverage -> we want export)